tests/echo/host/host.c

```diff
@@ -11,13 +11,13 @@
 #include "../../../host/strings.h"
 #include "echo_u.h"
 
-int host_echo(char* in, char* out, char* str1, char* str2, char str3[100])
+int host_echo(char* in, char* out, char* str1, char* str2, char* str3, int length)
```

Comment:
Just a suggestion: rename length to out_length to make it clearer what it's the length of.